docs/development.md

```diff
@@ -58,6 +58,16 @@ $ chown -R $USER docs/reference/build  # The generated docs are owned by root.
 $ kubebuilder create config --crds --output cluster-registry-crd.yaml
 ```
 
+**NOTE:** If you want to use `cluster-registry-crd.yaml` in a helm chart, then it is
+suggested to add the following annotation to `cluster-registry-crd.yaml`, this can
```

Comment:
Can you update the text a bit?

"... `cluster-registry-crd.yaml`. This ensures that the cluster registry CRD is created before other resources in the Helm chart. This annotation is available in Helm 2.10+."

Reply:
Done. Thanks @perotinus